### PR TITLE
Added cfn template and lambda code for alert

### PR DESCRIPTION
**Pull request by skzmego**

### CODE REVIEW

File: alert-notification/lambda_function.py [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+                    {"type": "mrkdwn", "text": "*Timestamp:*\n{}".format(timestamp)},
+                ],
+            },
+            {
+                "type": "actions",
+                "elements": [
+                    {
+                        "type": "button",
+                        "text": {"type": "plain_text", "text": "View Alarm"},
+                        "value": "Cloudwatch Alarm-AWS",
```
Comment (vaibhavqrcg) [inline alert-notification/lambda_function.py]: See if after removeing the action_id & value, the triangle issue is gone. I'm hopeful.

**Review by vaibhavqrcg — CHANGES_REQUESTED** [review]

File: alert-notification/lambda_function.py [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+
+    req = Request(
+        "https://slack.com/api/chat.postMessage",
+        headers=headers,
+        data=json.dumps(data).encode("utf-8"),
+        method="POST",
+    )
+
+    response = json.loads(urlopen(req).read().decode("utf-8"))
+    print(response)
```
Comment (vaibhavqrcg) [inline alert-notification/lambda_function.py]: Can we also remove debug statements?

File: alert-notification/lambda_function.py [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+    data = {"channel": channel_id, "blocks": query_data, "thread_ts": message_ts}
+
+    req = Request(
+        "https://slack.com/api/chat.postMessage",
+        headers=headers,
+        data=json.dumps(data).encode("utf-8"),
+        method="POST",
+    )
+    response = json.loads(urlopen(req).read().decode("utf-8"))
+    print(response)
```
Comment (vaibhavqrcg) [inline alert-notification/lambda_function.py]: Remove this too.

File: alert-notification/lambda_function.py (hunk 1 above)
Comment (skzmego) [reply alert-notification/lambda_function.py]: No luck 
image

File: alert-notification/lambda_function.py (hunk 2 above)
Comment (vaibhavqrcg) [reply alert-notification/lambda_function.py]: @skzmego we need to remove this?

File: alert-notification/lambda_function.py (hunk 3 above)
Comment (vaibhavqrcg) [reply alert-notification/lambda_function.py]: This as well.

**Review by vaibhavqrcg — CHANGES_REQUESTED** [review]

**Comment by vaibhavqrcg** [comment]
Should include a README file.

**Review by vaibhavqrcg — APPROVED** [review]
Good to merge with a README

File: alert-notification/lambda_function.py (hunk 2 above)
Comment (vaibhavqrcg) [reply alert-notification/lambda_function.py]: Yes, no debug statements.

**Pull request merged**